css-grid-classes/styles.css

```diff
@@ -0,0 +1,30 @@
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  align-items: center;
+}
+
+body {
+  font-family: sans-serif;
+}
+
+.column-one-half > p {
+  width: 33rem;
+}
+
+.column-one-half > img {
+  width: 90%;
+  height: 90%;
+}
+
+.column-one-fourth > img {
+  width: 100%;
+}
```

Comment:
This is not the goal of a grid system. You don't want to size the content inside of a column, you want to size the column itself, and just add content to that column in the HTML document.